classes/Configuration.php

```diff
@@ -1,4 +1,5 @@
 <?php
+
```

Comment:
Remove this line and we are good to go

Reply:
Hello, 
I just removed the line in my latest commit.